trezor-crypto/crypto/tests/test_check.c

```diff
@@ -2281,3067 +4024,5065 @@ END_TEST
+       "1376c6837f131f6cd1a45b1056297d2314aa0ac5f7d581d2d878261eb3259b4d",
+       "ce790760ada87dd819b59e4f6765d836d346567ec34f02bbcfcae0585c1d758f"},
+      {"cf209db9e7ee85f1e648924ec97edd86b56a833b25707519d4fbe64fd50e150a",
+       "804f0806087dc665a26230ed5fd44c062980ee182a6bd7dbdb33df018c983778",
+       "30d3c448cb08935309753b3051366f52328ca1d9a0b63c72b989edee0da32b0e",
+       "98e3c973a7e85b5eab8111521c66ca584bed5597f060ab0c6b5cdeece502ac48",
+       "2646276e1305396a1b2473690066011a39789570a09e10ce1a013c8f32cd5bea"},
+      {"b0a0ffeea67b656c4c585ba58ff528a6f45d2f915db98e4a14a8ff17f27fc105",
+       "4fabe16274f6af526ee053028485db6acd13804e02dcdddccc4183a319ab9e1c",
+       "1e140bb08a936ac6b7437644ca0769f3c165c7aa5501d49f064a0346179b4008",
+       "68fc1be64fb68761542a655b8dbebf50980f1fbc1845528df8d8a06bf89a1495",
+       "7dab86994b47014efe38493fc2b62ffcead806da6e0d73c992db8cb5618a19dc"},
+      {"0fee422c2294b06ca83bc3704384dffc580e7ff5921881e51a755e5f9b80af03",
+       "4359a663ead3f7ffc3a0ead5c3c2bde348017e7bfa620f21759c32e469a16dfe",
+       "532066e3eec29334fffc37b17178dfbac3bee15f7845f01449ddbaf5e57a7b0c",
+       "32e46c2fb99402837631c8175db31cdd334c145f922be9070d62e6d9c493c3ea",
+       "8c7b7d2d61cdb648960434d894787426a76d16dd46949c7aa4b85dcf1054b4d5"},
+      {"3a712d5b7ceb5257dcf6e6bb06548de6ef3deba5d456cd91fc305a12b46b5d01",
+       "5e7da62e3ec42cf3e554639dd4d2006754ee6839b720cadba94a26b73b1665ee",
+       "2a518ecab17a2d9dde219c775bcf4f2306b190bef2dea34fb65b8e4dccc13405",
+       "3b5d66a4dfb068923b3bc21cc8b40b59e12f845e0b85a86d394db0fa310bf185",
+       "2ec17f1cc0be093e9cdb741a991c0f417230dea275cd7babdad35e949e250521"},
+      {"5f815f2d65cef584c5e5d48b2d3d3e4cae310d70b328f88af6e9f63c52b4c90d",
+       "8a539a8c6b2339922b31cf4bc064f1fedeb3912fd89585d79dfcff2a60aee295",
+       "385f7132b72db04146b9e472736b32adfca29556b4775a743c18e2bfab939007",
+       "884aaf96d625968ddb2582922a87abca131272884c47f6b86890ebccf0a79d5b",
+       "a7afdaf24fe8472d8b89e95c3ce4a40bdf700af7cedee44ed3aa5ccca09839bd"},
+      {"a043340d072df16a8ab5135f8c1d601bff14c5aba01b9212b886ad71fe164506",
+       "52f6de5fa0fae32d4020a54d395319509d6b92092a0bf849fb34e73f8e71fc99",
+       "37d7472d360164da29e6dcb8f9796976022571c5df4ddf7e30e9a579ba13d509",
+       "8c369e3fd5b1112e4437b1f09e987acca4966f2f8c5227eb15ace240a2c64cc7",
+       "fc795fe7baff5c3ac98366e6882f25874ea2b0a649d16f139e5c54ea47042a1e"},
+      {"97a3268db03fa184c8cba020bf216fc789292fa9615a28962385b86870ffd70f",
+       "a76c215587022bb9252ece4c5afeb0e65b820834cd41ac76e6c062d3eea75dc6",
+       "8310271017154cbddf7005e24eb9a9a86777b3f42fa5e35095eafaac4eb24802",
+       "b822665c2406083c851ecaa91ea67aa740c057e7679b5755cee60a6c63f17fd6",
+       "f83e2444527056eba595d49bde40b2e8da76d2c145f203331d26e94560993fbc"},
+      {"edaad13efad39f26298e86ba8d06a46e59122232c9529bd22f2f656595421e00",
+       "f38e56a79f5159eb3b581dea537ec12c9c6fac381b2cf6073e27fc621197cb62",
+    
```

Comment:
`test_schnorr_sign_verify` was removed, this test was also from wallet core

Reply:
Re-added